sdk/servicebus/service-bus/test/receiveAndDeleteMode.spec.ts

```diff
@@ -414,7 +414,7 @@ describe("Unsupported features in ReceiveAndDelete mode", function(): void {
     return msgs[0];
   }
 
-  const testError = (err: Error, operation: DispositionType) => {
+  const testError = (err: Error, operation: DispositionType): void => {
```

Comment:
@ramya-rao-a what's the appropriate type for testError ? 'onError' doesn't work

Reply:
void is right